source/reference/minio-mc-admin/mc-admin-user.rst

```diff
@@ -22,10 +22,10 @@ The :mc:`mc admin user` command and its subcommands manage :ref:`MinIO users <mi
 Clients *must* authenticate to the MinIO deployment with the access key and secret key associated to a user on the deployment.
 MinIO users constitute a key component in MinIO Identity and Access Management.
 
-To manage users who authenticate using a 3rd party IDP, use the :mc:`mc admin` commands for the appropriate provider:
+To manage users who authenticate using a 3rd party IDP, use the :mc:`mc` commands for the appropriate provider:
```

Comment:
I could make this work if it seems important:
```
:mc:`mc idp`
```
But at the moment I went with altering the text. The links for ldap/openid are right there below this.